tool/embedded-binaries.ts

```diff
@@ -19,7 +20,10 @@ export async function getEmbeddedProtocol(outPath: string): Promise<void> {
   console.log('Downloading Embedded Sass Protocol.');
   try {
     const response = await fetch(
-      `https://raw.githubusercontent.com/sass/embedded-protocol/master/${protoName}`
+      `https://raw.githubusercontent.com/sass/embedded-protocol/master/${protoName}`,
+      {
+        redirect: 'follow' as RequestRedirect,
```

Comment:
I think this `as` shouldn't be necessary... `as` asserts that an expression of a broader type is in fact a narrower type, but `'follow'` is statically known to be a member of `RequestRedirect`. Similar comment below.